bindings/core/src/method/wallet.rs

```diff
@@ -129,17 +84,14 @@ pub enum WalletMethod {
         #[derivative(Debug(format_with = "OmittedDebug::omitted_fmt"))]
         password: String,
         /// If ignore_if_coin_type_mismatch.is_some(), client options will not be restored.
-        /// If ignore_if_coin_type_mismatch == Some(true), client options coin type and accounts will not be restored
+        /// If ignore_if_coin_type_mismatch == Some(true), client options coin type and wallet will not be restored
         /// if the cointype doesn't match.
         ignore_if_coin_type_mismatch: Option<bool>,
-        /// If ignore_if_bech32_hrp_mismatch == Some("rms"), but addresses have something different like "smr", no
-        /// accounts will be restored.
+        /// If ignore_if_bech32_hrp_mismatch == Some("rms"), but addresses have something different like "smr", the
```

Comment:
weird comment

Reply:
it isn't mine ... I just replaced a word.

Reply:
yeah, I know, but regardless it is weird and it's not difficult to fix these things while you touch the code around them